### DISCUSSION TITLE
IndexError: index is out of bounds

### DISCUSSION THREAD

**Discussion by Exo1943** [oobabooga/text-generation-webui#3275] [Q&A]
![Untitled](https://github.com/oobabooga/text-generation-webui/assets/140352176/fe9de4ae-ce26-48a5-81d5-d33605aa750e)

I am still new to using this program, so I am not sure what this means.  I do not what to edit the python script without asking first, because I have no clue what this line does.

This is the line of code in context:

![2](https://github.com/oobabooga/text-generation-webui/assets/140352176/6fb793e3-e054-4e15-a4a5-8e304eb72532)

**Comment by knight-captain** [comment]
I got this exact error. I was able to fix it by running the update_windows.bat, and then Migrating to the newer installer as per https://github.com/oobabooga/text-generation-webui/wiki/Migrating-an-old-one%E2%80%90click-install
